cashkick-frontend/src/components/atoms/Icon/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode, ReactElement, ReactPortal } from "react";
+import { SvgIcon } from '@mui/material';
+
+export interface IconProps {
+  iconPath?: string | ReactNode; 
+  cssDesign?: string;  
+};
+
+const Icon: React.FC<IconProps> = ({ iconPath, cssDesign }) => {
+
+  if (React.isValidElement(iconPath)) {
+    return iconPath;
+  }
+
+  if (typeof iconPath === "string") {
```

Comment:
you can just do it this way - (typeof iconPath === "string") && <>your component</>